Mathlib/Data/List/Chain.lean

```diff
@@ -419,6 +419,30 @@ lemma Chain'.iterate_eq_of_apply_eq {α : Type*} {f : α → α} {l : List α}
     apply hl
     omega
 
+theorem chain_replicate [IsRefl α r] (n : ℕ) (a : α) : Chain r a (replicate n a) :=
```

Comment:
Maybe `chain_replicate_of_isRefl`?

Reply:
Reflexive relations are precisely the ones for which `replicate` is always `chain'`, so I'm not sure there's a need to disambiguate.